docs/devenv.rst

```diff
@@ -206,6 +206,20 @@ includes are defined in ``setup.cfg``.
 To upgrade the development tools in the future, just rerun the exact same
 command.
 
+.. tip::
+
+    This command often fails on a mac, in that case please run::
```

Comment:
I don't think there's any "often" about this. It just depends if you have all the dependencies installed.

We try to avoid littering generic docs with platform specific installation instructions. We do need to address the fact that everyone (mac or otherwise) needs the non-Python graphviz dependencies in order to build the documentation. I'm not sure where best to put it.